wix/Build.OpenJDK_generic.cmd

```diff
@@ -76,16 +78,16 @@ SET JEP322_BASE_NAME="unknown"
 IF %PRODUCT_CATEGORY% EQU jre SET JRE=-jre
 IF %PRODUCT_MAJOR_VERSION% GEQ 10 (
     IF %PRODUCT_MAJOR_VERSION% EQU 11 SET LTS_VERSION=-LTS
-    IF DEFINED PRODUCT_PATCH_VERSION (
-        SET PATCH=+%PRODUCT_PATCH_VERSION%
+    IF DEFINED PRODUCT_BUILD_NUMBER (
```

Comment:
Looks like we only set the build number if a product build number is passed in.  What if only a patch version is passed in - do we see identical behaviour as we did in releases before 11.0.9.1?

Reply:
LGTM, If we pass a PRODUCT_BUILD_NUMBER previously (erroneously) known as PRODUCT_PATCH_VERSION
we try to add "+<id>".
If not build number .. no "+" appended